### PR TITLE
Added result set and functional generator

### PR DESCRIPTION
**Pull request by yasha-dev1**
**Checklist**
- [x] My code follows the style guidelines of this project
- [x] My commits follow the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: includes/api/class-urlslab-api-keywords.php [hunk 1]
```diff
@@ -491,8 +491,6 @@ protected function get_items_sql( WP_REST_Request $request ): Urlslab_Api_Table_
 			)
 		);
 
-		$sql->add_group_by( 'q.query_id' );
-
```
Comment (vzeman) [inline includes/api/class-urlslab-api-keywords.php:495]: this was done in another PR

File: urlslab.php [hunk 2]
```diff
@@ -16,7 +16,7 @@
  * Plugin Name:       URLsLab
  * Plugin URI:        https://github.com/QualityUnit/wp-urlslab
  * Description:       URLsLab WordPress Plugin to optimize your website for search engines and enhance automatically content
- * Version: 2.60.0-beta.4
+ * Version: 2.60.1
```
Comment (vzeman) [inline urlslab.php:19]: change to 2.61.0

**Review by vzeman — APPROVED** [review]

**Review by mvstepanek — APPROVED** [review]
